snapcraft.yaml

```diff
@@ -60,9 +60,15 @@ website: https://ubuntu.com/lxd
 confinement: strict
 
 plugs:
- ceph-conf:
-   interface: content
-   target: "$SNAP_DATA/microceph"
+  ceph-conf:
+    interface: content
+    target: "$SNAP_DATA/microceph"
+  nvme-subsystem:
+    interface: system-files
+    read:
+      - /sys/class/nvme-subsystem
+    write:
+      - /sys/class/nvme-subsystem
```

Comment:
According to the [doc](https://snapcraft.io/docs/system-files-interface), `write` grants both `read` and `write`.

Reply:
Thanks, I have now removed `write` completely since only read access to the path is required.

Reply:
Ah, even better! Thanks

Reply:
Now the extra plug is completely removed since the access is already provided by the base template. Some more details here: https://chat.charmhub.io/charmhub/pl/hi6qo7gaetd9dkwxe3gph9o8xa